client/src/utils/URLpath.js

```diff
@@ -1,5 +1,7 @@
+export const aiEndpoint = process.env.REACT_APP_AI_KEY || 'https://aii.littersort.ai';
+
 export default function URLpath(endpoint, extension) {
-  const deployed = process.env.REACT_APP_API_KEY;
+  const deployed = process.env.REACT_APP_API_KEY || 'https://api.littersort.ai';
```

Comment:
if not set just ping prod? hmmmm seems risky imo easy setup for being attacked

Reply:
I could be overthinking it, just marking here for now, no need to fix until confirmed and can be in another pr

Reply:
There were issues connecting the env file to the frontend and this was the workaround to get everything running for now. This is a temporary solution and we will remove the direct links once we figure out what is going wrong.

Reply:
The direct links are no longer needed so they have been removed.